lib/logger/test/logger/translator_test.exs

```diff
@@ -530,6 +530,34 @@ defmodule Logger.TranslatorTest do
     assert {%RuntimeError{message: "oops"}, [_ | _]} = process_metadata[:crash_reason]
   end
 
+  if :erlang.system_info(:otp_release) >= ~c"27" do
```

Comment:
We got it too 🙃 

```suggestion
  if System.otp_release() >= "27" do
```

Reply:
I like to do the following:

```elixir
@tag skip: System.otp_release() < "27"
```

(or `if ..., do: @tag :skip`.)

because it's less indent and less git diff when it's ultimately gonna be enabled. :)

Reply:
Sure also works!

Reply:
Oh that's neat! Thanks @whatyouhide @wojtekmach !